docs/DeveloperGuide.adoc

```diff
@@ -297,15 +297,20 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new note | Take note of details that I may use later
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |delete a note | Remove note that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |edit existing note| Correct any typo or add more details
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |search by keyword | Locate details of notes without having to go through the entire directory
+
+|`* *` |user |command suggestion | Minimize the chance of mistyping a command
+
+|`* *` |user |quick fill| Save time typing out the full command
+
+|`*` |user |export to pdf | Print the notes to bring it into an examination hall
```

Comment:
Let's add in the stories from [over here](https://docs.google.com/document/d/1c2IjdAFxckrz8-HlwbnpFfjcb_gzGKt2mtYCQhZwahI/edit)?

Reply:
Perhaps you could filter out the stories from the docs to include only those that are relevant with our new development (the parts that are currently planned to be developed)